tests/src/chat-signals.ts

```diff
@@ -63,14 +63,31 @@ module.exports = async (orchestrator) => {
 
     // waiting for the signal to be received by bob.
     for (let i = 0; i < 5; i++) {
-      if (flag) break;
-      console.log(`wating for signal: ${i}`)
+      if (receivedCount > 0) break;
+      console.log(`waiting for signal: ${i}`)
       await delay(500)
     }
-    t.ok(flag)
+    t.equal(receivedCount, 1)
 
     stats = await alice_chat.call('chat', 'stats', {category: "General"});
     t.deepEqual(stats, {agents: 2, active: 2, channels: 1, messages: 1});
 
+    const r5 = await alice_chat.call('chat', 'signal_specific_chatters', {
```

Comment:
would be nice if this test include a third agent who we test does NOT receive this signal